### DISCUSSION TITLE
Calculating order totals

### DISCUSSION THREAD

**Discussion by amacieli** [Budibase/budibase#8368] [Help]
I have a price list table (Budibase DB) with one row per item, and its price.
I have a form where you can enter the number of each type of item.
On that form, I would like to have it calculate the order total as the form is filled in.
Two-part question:
a) how can I look up the price in the price list table, and include in a calculation?
b) how can I run the same calculation script after each form field?
I looked through the discussions, but couldn't find anything (that I thought) was relevant...
Thanks!!

**Comment by melohagan** [comment]
Hey @amacieli 

Does this tutorial help? https://docs.budibase.com/docs/multi-select-picker#tutorial-sum-of-selected-items
